docs/migration/migrate_8_0.asciidoc

```diff
@@ -118,7 +118,7 @@ logging:
 
 [float]
 ==== Set log verbosity with root
-*Details:* Previously logging output would be specified by `logging.silent` (none), `logging.quiet` (error messages only) and `logging.verbose` (all). With the new logging configuration, set the minimum required log level.
+*Details:* Previously logging output would be specified by `logging.silent` (none), `logging.quiet` (error messages only) and `logging.verbose` (all). With the new logging configuration, set the minimum required {kibana-ref}/logging-service.html#log-level[log level].
```

Comment:
question: Is it possible to use the standard `asciidoc` cross-references instead of relying on generated URL? 
https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#ex-xrefs
Should the file name change, the link is broken.

Reply:
The doc we're linking to isn't in the same book so we have to use an external link here.